jax/_src/scipy/stats/_core.py

```diff
@@ -207,9 +207,9 @@ def sem(a: ArrayLike, axis: int | None = 0, ddof: int = 1, nan_policy: str = "pr
     b = b.ravel()
```

Comment:
You need to remove this block if you want to push this logic into the reductions below here.

Reply:
Thanks! Removed.